### PR TITLE
Concatenation of adt's into 1 ObsSpace, cp0.b QC for adt and sst

### LINKED ISSUE #597: Update QC and obs ingest
- concatenate altimeters into 1 obs space to better estimate the offset between obs and model
- update the QC flags for adt and sst's following what was done for `cp0.b`

### PR DESCRIPTION
**Pull request by guillaumevernieres**
## New QC parameters
- Update of the SST QC following what was done for `cp0.b`. 
- Altimeters from the RADS can be safely concatenated into 1 obs space, that should have been done ages ago! 

- fixes #597

### CODE REVIEW

File: parm/soca/obs/config/sst_metopa_l3u_so025.yaml
Comment (ShastriPaturi) [inline parm/soca/obs/config/sst_metopa_l3u_so025.yaml:1]: Where are the values of minvalue coming from?

**Review by ShastriPaturi — CHANGES_REQUESTED** [review]

File: parm/soca/obs/config/sst_metopa_l3u_so025.yaml
Comment (guillaumevernieres) [reply parm/soca/obs/config/sst_metopa_l3u_so025.yaml:1]: min value for what @ShastriPaturi ?

Comment (ShastriPaturi) [reply parm/soca/obs/config/sst_metopa_l3u_so025.yaml:1]: obsError/seaSurfaceTemperature etc in the sst*yaml file.

**Comment by guillaumevernieres** [comment]
An unfortunate ioda PR:
https://github.com/JCSDA-internal/ioda/pull/1077/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR58

**Comment by hyunchul386** [comment]
For me, the changes in SST filters and background errors looks fine.

**Review by hyunchul386 — DISMISSED** [review]

**Review by ShastriPaturi — APPROVED** [review]
Looks Good

**Pull request merged**